nomad/structs/structs.go

```diff
@@ -5553,6 +5569,37 @@ func (t *Task) Warnings() error {
 	return mErr.ErrorOrNil()
 }
 
+type Kind string
+
+const connect_prefix = "connect:"
```

Comment:
We decided in the RFC that this should be `connect-proxy` incase we end up with other kinds like `connect-gateway`.

Reply:
Great catch! :grimacing: